src/views/Home/components/Milestones/milestones.scss

```diff
@@ -1,3 +1,81 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/typography";
+@use "~stylesheets/responsiveness";
 
-/* Delete these lines when you begin development */
+.Milestones {
+  overflow: auto;
+  margin: 0;
+  min-height: 375px;
+  background: url("~images/components/milestones/background.png") repeat-y, var(--dark-purple);
```

Comment:
let's not condense the component background into a single png. consider pulling out the grid & lens flare sphere SVGs from Figma & layering them with the accent graphics in the container mentioned in my other comment.

Reply:
I can't find a way to repeat a SVG vertically based on height without using the background property. Though, I replaced the png with svg in my newer commit. 

Let me know if there's a better way!

Reply:
@mocklee Continuing https://github.com/CruzHacks/cruzhacks-2021-website/pull/97#discussion_r539613066, `preserveAspectRatio="none"` doesn't quite work because it would stretch the grid. I'm using `preserveAspectRatio="xMinYMax slice"` for now so the bottom would stay consistent. The top would be cropped in some cases but that's the best that I can do. Please let me know if I'm missing anything.

Reply:
yea `xMinYMax slice` is a pretty clean solution, grid lines are indeed tricky.. 

this implementation of the background grid SVG is probably as good as it'll get without tossing out SVG altogether & drawing responsive squares with CSS (which we probably don't have time to do 😅️)